packages/fhir-converter/src/templates/cda/Sections/HealthConcerns.hbs

```diff
@@ -1,7 +1,7 @@
 {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.58')}}
     {{#each (toArray 2_16_840_1_113883_10_20_22_2_58.entry)}}
         {{#each (toArray this.act.entryRelationship) as |healthEntry|}}
-            {{#if healthEntry.observation}}
+            {{#if healthEntry.observation.value.code}}
```

Comment:
only create this observation if there is a value code since they have a tendency to just be kinda empty